src/shared/router.ts

```diff
@@ -1,38 +1,38 @@
-import { App, createApp } from 'vue';
+import { App, createApp } from "vue";
 
 let previousRoute = "";
-let callbackRoute = (url:string) => {};
-let apps:Array<App<Element>> = [];
+let callbackRoute = (url: string) => {};
+const apps: Array<App<Element>> = [];
```

Comment:
Just wondering, why is this one of the only `Array<>` typings that is not picked up by the rules?

Reply:
would you know what rule does that @bartduisters? :sweat_smile:

Reply:
[the rule](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/array-type.md)
☝️ this with the value `generic`

Reply:
I mean, that rule with the value `generic` will always enforce `Array<T>`, which is what I suggested